src/test/span_tests.cpp

```diff
@@ -64,7 +64,7 @@ BOOST_AUTO_TEST_CASE(span_constructor_sfinae)
     BOOST_CHECK(!Spannable(std::set<int>{}));
     BOOST_CHECK(!Spannable(std::vector<bool>{}));
     BOOST_CHECK(Spannable(std::array<int, 3>{}));
-    BOOST_CHECK(Spannable(Span<int>{}));
+    BOOST_CHECK(Spannable(std::span<int>{}));
     BOOST_CHECK(Spannable("char array"));
     BOOST_CHECK(Spannable(SpannableYes{}));
```

Comment:
It seems to me that `SpannableYes` needs a `begin`/`end` now, not a `data`/`size` (`SpannableNo` should also be updated accordingly)
```C++
struct SpannableYes
{
    const char* begin();
    const char* end();
};
```